WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -101,7 +107,14 @@ class BlogDashboardViewModel {
     }
 
     func isQuickActionsSection(_ sectionIndex: Int) -> Bool {
-        return sectionIndex == DashboardSection.quickActions.rawValue
+        let showMigration = MigrationSuccessCardView.shouldShowMigrationSuccessCard && !WPDeviceIdentification.isiPad()
+        let targetIndex = showMigration ? DashboardSection.quickActions.rawValue : DashboardSection.quickActions.rawValue - 1
```

Comment:
This is a bit tricky, but I also assume the migration success card section would be removed before any new sections are added to the view. Do you think that's the case?

Reply:
Exactly, there will be changes to the logic that shows the migration, so anything related to showing/not showing will evolve in future PRs (same applies to other comments that refer to conditional logic related to the success card)